gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,419 @@
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "New Repositories Created ",
+            "type": "timeseries",
+            "width": 43,
+            "x": 79,
+            "y": 45
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "18",
+            "height": 10,
+            "html": "Project Information",
+            "id": 10,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 43,
+            "x": 79,
+            "y": 1
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "18",
+            "height": 8,
+            "html": "Users",
+            "id": 11,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 31,
+            "x": 1,
+            "y": 39
+        },
+        {
+            "height": 13,
+            "id": 12,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.user_session_logins_total{*}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "User Logins ",
+            "type": "timeseries",
+            "width": 31,
+            "x": 1,
+            "y": 49
+        },
+        {
+            "height": 13,
+            "id": 13,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.unicorn.workers{*} by {host}",
```

Comment:
- ditto template variables
- why grouping by `host`? This will conflicts with the template variables